eng/Versions.props

```diff
@@ -7,7 +7,7 @@
     <MinorVersion>0</MinorVersion>
     <PatchVersion>0</PatchVersion>
     <SdkBandVersion>9.0.100</SdkBandVersion>
-    <PackageVersionNet8>8.0.11</PackageVersionNet8>
+    <PackageVersionNet8>8.0.$([MSBuild]::Add($([System.Version]::Parse('$(ProductVersion)').Build),11))</PackageVersionNet8>
```

Comment:
I think you want PatchVersion here and not ProductVersion

Reply:
It parses the ProductVersion, at lest in 8.0 that also changed?

Reply:
ahh, I just saw the productversion and didn't notice the parse. Does parsing that and grabbing build do the right thing? I actually don't know.